scripts/s3_media_upload

```diff
@@ -406,6 +420,8 @@ def main():
     )
 
     args = parser.parse_args()
+    if args.no_progress:
+        progress=False
```

Comment:
Don't you need a `global progress`?

Reply:
Apparently only if you want the ability to not specify `--no-progress` is this important; looks like the current matrix-org-hotfixes just doesn't support progress bars at all.